pinot-spi/src/main/java/org/apache/pinot/spi/config/table/FieldConfig.java

```diff
@@ -44,6 +44,9 @@
   // the cache improves performance of repeatable queries
   public static String TEXT_INDEX_ENABLE_QUERY_CACHE = "enableQueryCacheForTextIndex";
   public static String TEXT_INDEX_USE_AND_FOR_MULTI_TERM_QUERIES = "useANDForMultiTermTextIndexQueries";
+  public static String TEXT_INDEX_NO_RAW_DATA = "noRawDataForTextIndex";
+  public static String TEXT_INDEX_RAW_VALUE = "rawValueForTextIndex";
+  public static String TEXT_INDEX_DEFAULT_RAW_VALUE = "null";
```

Comment:
Does it make sense to use a single character for this? This value is never used in processing, so it is a pure memory overhead, right?

Reply:
using "n" as default